scripts/partition_manager.py

```diff
@@ -62,7 +62,7 @@ def empty_one_of(one_of_list):
                 reqs[k] = [i if i not in to_remove else to_add.pop(0) for i in v]
 
 
-def remove_all_zero_sized_partitions(reqs, to_delete=None):
+def remove_all_zero_sized_partitions(reqs, dp='app', to_delete=None):
```

Comment:
Having a default `dp='app'` seems vaguely like a bad idea :P (not just here). I think it could give annoying bugs at some point.

Reply:
I agree, should be fixed now.